docs/rules/escape-case.md

```diff
@@ -0,0 +1,22 @@
+# Enforce escape sequences to use uppercase values
+
+Enforces a convention of defining escape sequences with uppercase characters rather than lowercase ones.
+
```

Comment:
Should include reasoning for why this rule exists. Basically that it's more readable to have the value uppercase so it's distinct from the escape identifier.